BlazorAccessibilityTests.Tests/AriaAccordionTests.razor

```diff
@@ -1,335 +1,197 @@
-    }
-
-// region		div	
-// Creates a landmark region that contains the currently expanded accordion panel.
-
-    [Fact(DisplayName = "Creates a landmark region that contains the currently expanded accordion panel.")]
-    public void RegionAttribute()
-    {
-        var cut = Render(@<AriaAccordion>
-                                <AccordionPanel Title="P1">
-                                    <p>P1 Content</p>
-                                </AccordionPanel>
-                                <AccordionPanel Title="P2">
-                                    <p>P2 Content</p>
-                                </AccordionPanel>
-                        </AriaAccordion>
-    );
+        RenderTree.Add<AriaAccordion>();
 
-        var panels = cut.FindComponents<AccordionPanel>();
-        var content1 = panels[0].Find("div");
-        var content2 = panels[1].Find("div");
-
-        content1.GetAttribute("role").Should().Be("region");
-        content2.HasAttribute("role").Should().BeFalse();
+        var cut = Render(@<AccordionPanel Title="P1">CONTENT</AccordionPanel>);
+        
+        var buttonControlsId = cut.Find("button").GetAttribute("aria-controls");
+        cut.Find("div").Id.Should().Be(buttonControlsId);            
     }
 
-// aria-labelledby="IDREF"	div	
-// Defines the accessible name for the region element.
-// References the accordion header button that expands and collapses the region.
-// region elements are required to have an accessible name to be identified as a landmark.
-
+    // aria-labelledby="IDREF"	div	
+    // Defines the accessible name for the region element.
+    // References the accordion header button that expands and collapses the region.
+    // region elements are required to have an accessible name to be identified as a landmark.
     [Fact(DisplayName = "aria-labelledby=IDREF References the accordion header button that expands and collapses the region.")]
     public void AriaLabelledByAttribute()
     {
-        var cut = Render(@<AriaAccordion>
-                                <AccordionPanel Title="P1">
-                                    <p>P1 Content</p>
-                                </AccordionPanel>
-    </AriaAccordion>
-    );
-
-        var panel = cut.FindComponent<AccordionPanel>();
-        var buttonId = panel.Find("button").GetAttribute("id");
-
-        panel.Find("div")
-            .GetAttribute("aria-labelledby")
-            .Should().Be(buttonId);
-
-    }
-
-
-    // Tab	
-    // Moves focus to the next focusable element.
-    // All focusable elements in the accordion are included in the page Tab sequence.
-
-    // Shift + Tab	
-    // Moves focus to the previous focusable element.
-    // All focusable elements in the accordion are included in the page Tab sequence.
-
-    [Fact(
-        DisplayName = " All focusable elements in the accordion are included in the page Tab sequence.",
-        Skip = "USE e2e?")]
-    public void TabForward()
```

Comment:
I would actually not test this, not in e2e either, since this is browser behavior, and we can expect that to be correct (dont test other peoples code).

It is possible for us to change the normal tab order by adding `tabindex` attributes to elements. If we have done so, we can definitely write a test in bUnit that verifies these are set correctly.